src/cli.cr

```diff
@@ -40,6 +270,149 @@ CLI.run(
           },
         },
       },
+      add_user: {
+        info:      "add a user",
+        action:    "DppmRestApi::CLI.add_user",
```

Comment:
`DppmRestApi::CLI.` can be dropped (see also all other places below)

Reply:
It can just be `action: :add_user`?

Reply:
I tried setting the values like this and it said `undefined method 'run_server' for CLI:Module`

Reply:
That depends where the macro is expanded.
I think it can be inside the `CLI` module, inside `def run`, then call `CLI.run`. I've pushed a commit to change `CLI` to `DPPM::CLI`.

Reply:
I see. Ok.

Reply:
This breaks things. See the difference between [8aac32c2 and c1c4e02](https://github.com/DFabric/dppm-rest-api/pull/8/commits/8aac32c2718b9ae0eb8c3f508386288774de3abd). I think that's an issue with DPPM's not being namespaced?

Reply:
nevermind it works after running shards update

Reply:
Actually it doesn't.

`action: :run_server` gives `undefined method 'run_server' for DPPM::CLI:Module`

`action: run_server` or `action: DppmRestApi::CLI.run_server` gives `undefined macro method 'Call#split'`